GTG/gtk/errorhandler.py

```diff
@@ -24,25 +24,23 @@ def __init__(self, exception=None, main_msg=None, ignorable: bool = False, conte
 
         formatting = {
             'url': info.REPORT_BUG_URL,
-            'urlname': info.REPORT_BUG_URL,
-            'appname': info.NAME,
+            'urlname': _("our issue tracker"),
```

Comment:
Add a new line before it with a comment for the translators to have some context: `# Translators: Used as the link name in the error handler dialog (that goes to the issue tracker)`

Reply:
I actually hesitated in just hardcoding this right into the sentence's string, instead of keeping it as a variable. Or do we suspect distros might patch this and the URL to point to their bug tracker?

Reply:
If we use english there I am ok with embedding `urlname` in the description, but I'd keep the `url` separate so we (and distro maintainers/packagers) can change the issue URL without invalidating the translations.